app/controllers/feedbacks_controller.rb

```diff
@@ -7,7 +7,7 @@ def create
     @feedback = Feedback.create(feedback_params.merge(user: current_user, expert: current_expert))
     @current_roles = current_roles
     if @feedback.persisted?
-      @feedback.notify!
+      UserMailer.match_feedback(@feedback).deliver_now
```

Comment:
Euh, pas `deliver_now` je pense.
```suggestion
      UserMailer.delay.match_feedback(@feedback)
```

Reply:
Yes c'est un oublie quand j'ai voulu tester